### PR TITLE
Mlops add model v0

### PR DESCRIPTION
**Pull request by brunolajoie**
**MVP of taxifare_model package**

- Working code inside `model/taxifare_model` but ugly and in need of refacto.
- `main.py` contains training routes which can train on a limited number of rows
- `main_incremental.py` contains training routes which can train incrementally on an unlimited number of rows
- no challengify TAG yet

@gmanchon you don't need to review the code inside `model/taxifare_model` not any `model/notebooks`, but you can
- Check the other files diffs
- Try to `pip install -e .` from `model` folder?
- Then check that you can run `python taxifare_model/main.py` and `python taxifare_model/main_incremental.py`? Stop the execution of the second one, it takes 40min to run ATM :D 
- Merge if you good

### CODE REVIEW

**Comment by brunolajoie** [comment]
Sorry no need to review notebook either, they are all WIP

File: model/setup.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from setuptools import find_packages
+from setuptools import setup
+
+with open('requirements.txt') as f:
```
Comment (gmanchon) [inline model/setup.py:4]: should we commit to single or double quotes everywhere ? PEP8 does not make a recommandation (only to use double quotes for multiline strings)

personally I use double quotes but we can decide what we want...

```suggestion
with open("requirements.txt") as f:
```

**Comment by gmanchon** [comment]
@brunolajoie looks great, I did not dig into the `.gitignore` files, vscode `settings.json` or `tests` too much. The package installs correctly and the imports work 👍

**Review by gmanchon — CHANGES_REQUESTED** [review]

**Comment by gmanchon** [comment]
I trained from the `model` directory using `python -m taxifare_model.main` and `python -m taxifare_model.main_incremental`

File: model/tests/unittests/test_data.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: UTF-8 -*-
+
+# Import from standard library
+import os
+import taxifare_model
+import pandas as pd
+# Import from our lib
+from taxifare_model.lib import clean_data
+import pytest
```
[the remark this answers is not in the document]
Comment (brunolajoie) [reply model/tests/unittests/test_data.py:9]: Cool! Yes, will do later with 
image

**Comment by brunolajoie** [comment]
ok, sure, i'll yapf the whole thing later anyway, this was just an ugly but working MVP

**Comment by gmanchon** [comment]
pretty cool MVP and going to be awesome 🔥

**Pull request merged**